Bonsai.Harp.CF/SyringePumpEvent.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Bonsai.Expressions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reactive.Linq;
+using TResult = System.String;
+using System.ComponentModel;
+
+namespace Bonsai.Harp.CF
+{
+    public enum SyringePumpEventType: byte
+    {
+        /* Event: STEP_STATE */
+        Step = 0,
```

Comment:
By default all enums start at zero so I usually don't use explicit assignments unless they represent something in the hardware such as a register value.